pkg/cmd/ui/ui.go

```diff
@@ -86,18 +85,24 @@ func EnterImageRepoExternalRepository() string {
 }
 
 // EnterOutputPath allows the user to specify the path where the gitops configuration must reside locally in a UI prompt.
-func EnterOutputPath() string {
+func EnterOutputPath(originalPath string) string {
 	var outputPath string
-	prompt := &survey.Input{
-		Message: "Provide a path to write GitOps resources?",
-		Help:    "This is the path where the GitOps repository configuration is stored locally before you push it to the repository GitopsRepoURL",
-		Default: ".",
+	var err error
+	if originalPath == "" {
+		prompt := &survey.Input{
+			Message: "Provide a path to write GitOps resources?",
+			Help:    "This is the path where the GitOps repository configuration is stored locally before you push it to the repository GitopsRepoURL",
+			Default: ".",
+		}
+		err = survey.AskOne(prompt, &outputPath, nil)
```

Comment:
You could simplify this to `handleError(survey.AskOne(prompt, &outputPath, nil))`